pandas/conftest.py

```diff
@@ -1165,6 +1166,7 @@ def iris(datapath):
     timezone.utc,
     timezone(timedelta(hours=1)),
     timezone(timedelta(hours=-1), name="foo"),
+    zoneinfo.ZoneInfo("US/Pacific"),
```

Comment:
Good to add `zoneinfo.ZoneInfo("UTC")` to this fixture and the `utc_fixture`

Reply:
good idea, will do